### ISSUE TITLE
http doesnt work

### ISSUE THREAD

**Issue by KGT1** [TheFlow95/node-qbittorrent-api-v2#1]
Logs:
```
TypeError [ERR_INVALID_PROTOCOL]: Protocol "http:" not supported. Expected "https:"
    at new ClientRequest (_http_client.js:120:11)
    at Object.request (https.js:289:10)
at Promise (#Project#/node_modules/qbittorrent-api-v2/src/qbt.js:1302:21)
```

**Comment by TheFlow95** [comment]
Hi, thanks for you feedback!
Can you include the corresponding line of code? I'm guessing that you specified the wrong URL in the `connect` method. Try something like this: `api.connect('https://<domain_name>', '<username>', '<password>')`.

**Comment by LarsOlt** [comment]
Same problem here, https and http don't work. 

If I do https://localhost:8080 I get the following error, even tho my login data is correct:

Error: Client network socket disconnected before secure TLS connection was established
    at connResetException (internal/errors.js:613:14)
    at TLSSocket.onConnectEnd (_tls_wrap.js:1545:19)    at TLSSocket.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1226:12)  
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  code: 'ECONNRESET',
  path: null,
  host: 'localhost',
  port: 8080,
  localAddress: undefined
}
Error: Login failed with username: lars
    at Object.exports.connect (C:\Users\larso\Downloads\node_modules\qbittorrent-api-v2\src\qbt.js:878:9)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)

**Comment by KGT1** [comment]
@TheFlow95 sorry for taking so long.
The corresponding code is just the example:
```
const api = require('qbittorrent-api-v2');

api.connect('http://192.168.2.105:8080', 'admin', 'adminadmin')
    .then(qbt => {
        qbt.torrents()
            .then(torrents => {
                console.log(torrents)
            })
            .catch(err => {
                console.error(err)
            })
    })
    .catch(err => {
        console.error(err)
    })
```

**Comment by TheFlow95** [comment]
Hey guys @LarsOlt @KGT1 

The issue should now be solved in version 1.2.1.
Please update your npm package and let me know if it works so I can close the issue.

**Comment by TheFlow95** [comment]
Closing this issue for inactivity (bug should be resolved)
